docs/tutorials/basic/symbol.md

```diff
@@ -396,25 +332,16 @@ c.tojson() == c2.tojson()
 
 ## Customized Symbol
 
-Most operators such as `mx.sym.Convolution` and `mx.sym.Reshape` are implemented
-in C++ for better performance. MXNet also allows users to write new operators
-using any front-end language such as Python. It often makes the developing and
-debugging much easier. To implement an operator in Python, refer to
-[How to create new operators](http://mxnet.io/faq/new_op.html).
+Most operators such as [mx.sym.Convolution](https://mxnet.incubator.apache.org/api/python/symbol/symbol.html#mxnet.symbol.Convolution) and [mx.sym.Reshape](https://mxnet.incubator.apache.org/api/python/symbol/symbol.html#mxnet.symbol.reshape) are implemented in C++ for better performance. MXNet also allows users to write new operators using any front-end language such as Python. It often makes the developing and debugging much easier. To implement an operator in Python, refer to [How to create new operators](http://mxnet.io/faq/new_op.html).
```

Comment:
Why remove the in-line code rendering? (e.g. `code`)? Can still add a link to the in-line code.